chapter_preliminaries/ndarray.md

```diff
@@ -306,6 +340,10 @@ torch.tensor([[2, 1, 4, 3], [1, 2, 3, 4], [4, 3, 2, 1]])
 tf.constant([[2, 1, 4, 3], [1, 2, 3, 4], [4, 3, 2, 1]])
 ```
 
+```{.python .input}
```

Comment:
Missing `#@tab jax`

Reply:
The syntax has changed, please use `%%tab jax`.